app/src/main/java/com/kickstarter/viewmodels/RewardViewHolderViewModel.kt

```diff
@@ -226,7 +228,8 @@ interface RewardViewHolderViewModel {
             projectAndReward
                 .map { it.first }
```

Comment:
Filter before these maps, if I recall correctly the currency field might be null

Reply:
in this case its an == check so it will always return a boolean, so even if one is null (currentCurrency in this case) it will still return a boolean.  it.currency is always a string, the currentCurrency is nullable, but thats okay